python/ray/experimental/serve/handle.py

```diff
@@ -0,0 +1,57 @@
+import ray
+from ray.experimental import serve
+
+
+class RayServeHandle:
+    """A handle to service endpoint.
+
+    Invoking this endpoint with .remote is equivalent to pinging
+    a HTTP endpoint.
+
+    Example:
+       >>> handle = serve.get_handle("my_endpoint")
+       >>> handle
+       RayServeHandle(
+            Endpoint="my_endpoint",
+            URL="...",
+            Traffic=...
+       )
+       >>> handle.remote(my_request_content)
+       ObjectID(...)
+       >>> ray.get(handle.remote(...))
+       # result
+       >>> ray.get(handle.remote(let_it_crash_request))
+       # raises RayTaskError Exception
+    """
+
+    def __init__(self, router_handle, endpoint_name):
+        self.router_handle = router_handle
+        self.endpoint_name = endpoint_name
+
+    def remote(self, *args):
+        result_oid_bytes = ray.get(
+            self.router_handle.produce.remote(self.endpoint_name, *args))
+        return ray.ObjectID(result_oid_bytes)
+
+    def get_traffic_policy(self):
+        # Currently checks global state because we are sure handle and
```

Comment:
Whenever you comment "Currently..." you should probably indicate what is planned for the future to prevent others making incompatible changes